index.html

```diff
@@ -31,36 +31,36 @@
 
             <ul class="nav top-nav d-flex justify-content-end">
 
-                <!-- NEWS -->
-                <li class="nav-item">
-                    <a class="nav-link" href="pages/news.html">NEWS</a>
-                </li>
-                <!-- DOCS -->
-                <li class="nav-item">
-                    <a class="nav-link" href="https://www.uyuni-project.org/uyuni-docs/">DOCS</a>
-                </li>
                 <!-- Dropdown for available Uyuni versions -->
                 <li class="nav-item dropdown">
                     <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                        VERSION
+                        DOWNLOAD
                     </a>
                     <div class="dropdown-menu bg-dark" aria-labelledby="navbarDropdown">
-                        <a class="dropdown-item" href="pages/stable-version.html">Stable</a>
-                        <a class="dropdown-item" href="pages/devel-version.html">Develop</a>
+                        <a class="dropdown-item" href="/pages/stable-version.html">Stable</a>
```

Comment:
Having relative links allows testing the website locally without any kind of webserver. Please rollback to relative links.

Reply:
I have gone back to relative links but you should not test locally without a webserver, as the styles do not load right.

E. g. without a webserver:
![image](https://user-images.githubusercontent.com/50680111/82924033-111b8a00-9f7c-11ea-9dfc-4177b8d2c5f2.png)

With a webserver:
![image](https://user-images.githubusercontent.com/50680111/82924067-1aa4f200-9f7c-11ea-8451-01bfac92d4d1.png)

I suggest installing https://www.npmjs.com/package/http-server, it does not need any configuration.

Reply:
I think we should then fix whatever is broken on the style.

I never noticed this, but as far as I can tell all other stiles are working fine, and the problem is only with the news boxes, right?

Reply:
I fail to see anything broken in the stylesheet. The news boxes were the ones I immediately noticed, there might be more.